### PR TITLE
docs(GUI): explain why some drives are not bootable out of the box

### PR DESCRIPTION
**Pull request by jviotti**
The usual case is images requiring special treatment. We recommend the
users to use specialised tooling for those cases.

Signed-off-by: Juan Cruz Viotti jviotti@openmailbox.org

### CODE REVIEW

File: docs/USER-DOCUMENTATION.md [hunk 1]
```diff
@@ -3,6 +3,15 @@ Etcher User Documentation
 
 This document contains how-tos and FAQs oriented to Etcher users.
 
+Why is my drive not bootable?
+-----------------------------
+
+Etcher copies images to drives byte by byte, without doing any transformation
+to the final device, which means images that require special treatment to be
+made bootable, like Windows images, will not work out of the box. In these
+cases, the general advice is to use tooling specific to those kind of images,
```
Comment (lurch) [inline docs/USER-DOCUMENTATION.md]: Perhaps "software" would be better here than "tooling" ?

**Comment by lurch** [comment]
Is it worth also explaining that Etcher doesn't set up "persistent" partitions like Unetbootin can do? (or should that come under a separate PR?)

File: docs/USER-DOCUMENTATION.md (hunk 1 above)
Comment (jviotti) [inline docs/USER-DOCUMENTATION.md]: Fixed!

**Comment by alexandrosm** [comment]
you may want to include specific problematic cases, as well as the specific
tools? if we start listing them here, we will have them ready when we want
a list of what to include in our warnings feature.

-- 

*Alexandros Marinos*

Founder & CEO, Resin.io


+1 206-637-5498

@alexandrosm

**Comment by jviotti** [comment]
@alexandrosm Done! 

@lurch Good idea! I'll do it in a separate PR.

**Comment by lurch** [comment]
It also looks like "Windows 10 IoT Core Dashboard" is needed for writing Windows 10 IoT to a SD card
https://developer.microsoft.com/en-us/windows/iot/Docs/GetStarted/rpi3/sdcard/stable/GetStartedStep1.htm

**Pull request merged**